src/vs/workbench/contrib/tasks/electron-browser/task.contribution.ts

```diff
@@ -555,7 +555,7 @@ class TaskService extends Disposable implements ITaskService {
 		CommandsRegistry.registerCommand({
 			id: 'workbench.action.tasks.runTask',
 			handler: (accessor, arg) => {
-				this.runTaskCommand(arg);
+				this.runTaskCommand(Types.isArray(arg) ? arg[0] : arg);
```

Comment:
I do not understand why these changes are needed? The only other place where we have to pass an array instead of a single object is here: https://github.com/microsoft/vscode/blob/7bc29bf67c17608b11e23a8497c8f1de5a0922cf/src/vs/workbench/services/configurationResolver/browser/configurationResolverService.ts#L279

Reply:
Since commands in keybindings.json take an array, it seems likely to me that many commands are already written to take an array. This is the case for the terminate task command. Command inputs will be more useful if we don't require existing commands to be rewritten.